packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -332,11 +350,26 @@ class TextStyle extends Diagnosticable {
       decoration: decoration ?? this.decoration,
       decorationColor: decorationColor ?? this.decorationColor,
       decorationStyle: decorationStyle ?? this.decorationStyle,
+      debugLabel: newDebugLabel,
     );
   }
 
-  /// Creates a copy of this text style but with the numeric fields multiplied
-  /// by the given factors and then incremented by the given deltas.
+  static String _capDebugLabelLength(String label) {
+    const int maxLabelLength = 100;
```

Comment:
i'd probably remove this entire function but if you do keep it:

- use the same constant everywhere (here and in the assert above)
- remove excessive braces